microsite/data/plugins/github-workflows.yaml

```
@@ -12,4 +12,4 @@ tags:
   - cd
   - github
   - workflows
-addedDate: '2023-09-12'
+addedDate: '2023-10-27'
```

```suggestion
addedDate: '2023-09-12'
```

Keeping this one as is since that's when it was added